latest/PIT/pit_common.yaml

```
@@ -0,0 +1,188 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/PIT/pit_common-1.0.0
+
+title: All PIT common metadata
+description: |
+  This schema is the union of all Roman PIT common metadata, including
+  those found in the referenced pit_basic schema.
+type: object
+allOf:
+  - $ref: asdf://stsci.edu/datamodels/roman/schemas/PIT/pit_basic-1.0.0
+  - type: object
+    properties:
+      file_date:
+        title: File creation date
+        description: |
+          File creation date-time in ISO 8601 format with UTC time
+          scale.
+        tag: tag:stsci.edu:asdf/time/time-1.*
+      start_time:
+        title: Start time
+        description: |
+          Start time of data acquisition in ISO 8601 format.
+          tag: tag:stsci.edu:asdf/time/time-1.*
+      end_time:
+        title: End time
+        description: |
+          End time of data acquisition in ISO 8601 format.
+        tag: tag:stsci.edu:asdf/time/time-1.*
+      exposure_time:
+        # Optional
+        title: Exposure time (s)
+        description: |
+          The amount of time elapsed between the start and the end of
+          the observation or data stream.
+        type: number
+        unit: "s"
+      effective_exposure_time:
+        title: Effective exposure time (s)
+        description: |
+          The total accumulated effective exposure duration for this
+          data product, excluding dead time and lost time.
+        type: number
+        unit: "s"
+      instrument:
+        title: Instrument observing configuration
+        type: object
+        properties:
+          name:
+            title: Instrument name
+            anyOf:
+              - type: string
+              - type: "null"
+          detector:
+            title: Detector
+            anyOf:
+              - type: string
+              - type: "null"
+          optical_element:
+            title: Optical element
+            description: |
+              Name of filter, disperser, or other optical element.
+            anyOf:
+              - type: string
+              - type: "null"
+        required: [name, detector, optical_element]
+      telescope:
+        title: Telescope name
+        description: |
+          Name of telescope, typically Roman.
+        anyOf:
+          - type: string
+          - type: "null"
+      target_coordinates:
+        title: Position and region for this product
+        type: object
+        properties:
+          reference_frame:
+            anyOf:
+              - type: string
+              - type: "null"
```

```suggestion
          reference_frame:
            enum: [ICRS]
```
Can we force this to be ICRS to be consistent with SOC products?

---

Maybe `enum: ["ICRS", "N/A"]`, unless there's an easy/readable way to say like `type: "null" OR (type: string AND enum: ["ICRS"])`? Thinking about, say, the possibility of simulated or other data without coordinates.

---

We could use something like:
```yaml
anyOf:
  - enum: [ICRS]
  - type: "null"
```